NIP18: Reposts (#140)
nostr-protocol/nips: 18.md

NIP-18
======

Reposts
-------

`draft` `optional` `author:jb55`

> A repost is a `kind 6` note that is used to signal that you want your followers
> that another event is worth reading.

Reviewer: ```suggestion
A repost is a `kind 6` note that is used to signal to followers
that another event is worth reading.
```

The `content` of a repost event is empty.

The repost event MUST include an `e` tag with the `id` of the note that is

> being reposted. That tag SHOULD include a relay URL from where as its third entry
> that note can be fetched as its third entry.

Reviewer: ```suggestion
being reposted. That tag SHOULD include a relay URL as its third entry
to indicate where it can be fetched.
```

> The reaction SHOULD include a `p` tag with the `pubkey` of the event being

Reviewer 2: reaction?
